applications/projects/SofaGuiGlut/CMakeLists.txt

```diff
@@ -14,8 +15,6 @@ set(SOURCE_FILES
 
 list(APPEND SOURCE_FILES Main.cpp)
 
-find_package(GLUT REQUIRED)
```

Comment:
GLUT and Sofa.GL are not related so you must keep that

Reply:
Actually it was duplicated. See line 5 :arrow_up:

Reply:
ah ok 👍